src/t_string.c

```diff
@@ -719,11 +719,20 @@ void strlenCommand(client *c) {
  * STRALGO <algorithm> ... arguments ... */
 void stralgoLCS(client *c);     /* This implements the LCS algorithm. */
 void stralgoCommand(client *c) {
-    /* Select the algorithm. */
-    if (!strcasecmp(c->argv[1]->ptr,"lcs")) {
+    char *subcmd = c->argv[1]->ptr;
```

Comment:
I don't see the point of promoting this command to be a subcommand. I don't think we'll ever touch this again. It seems excessive to add a help here. This was one of salvatores pet projects.

Reply:
i with he didn't wrap LCS inside an STRALGO at all (and i also wish he would split it to two commands, one works on strings and one on keys).
At this point, splitting LCS into a sub-command may help a bit (to bring it closer to being a standalone command)

you're absolutely right that it's not really a sub-command like others (CLIENT, ACL, etc).
but in some way it's a bit similar to XGROUP and XINFO.

i don't have any conclusive opinion here.. it stinks either way.

Reply:
I get your point. It also is not that important, so I don't feel strongly either way. I would still prefer to just leave it as this weird command that has a weird required second argument (and maybe even update the docs to indicate as much).

Reply:
@madolson it's still that weird command but now it has two weird second arguments :)